lib/bot.js

```diff
@@ -92,6 +89,16 @@ const TwitchBot = class TwitchBot extends EventEmitter {
           this.emit('subscription', event)
         }
       }
+
+      if(data.includes('JOIN')) {
+        const channel = parser.formatJOIN(data)
+        this.emit('join',channel);
```

Comment:
space after `,`?

Reply:
Maybe configuring a .eslint-file that uses a certain styleguide might be a good idea so such things can be autocorrected.